src/network/webtask.cpp

```diff
@@ -262,14 +262,14 @@ void WebTask::slotAbort() {
         m_timeoutTimerId = kInvalidTimerId;
     }
 
-    auto* const pPendingNetworkReply = m_pendingNetworkReplyWeakPtr.data();
-    QUrl requestUrl;
-
     if (m_state == State::Looping) {
         DEBUG_ASSERT(m_state == State::Looping);
```

Comment:
The debug assert is redundant here, because the if condition with the same is just one line above.

Reply:
Done +1.